controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -642,6 +679,20 @@ func (r *StorageClassRequestReconciler) reconcileCephClientCephFSNode() error {
 	return nil
 }
 
+func (r *StorageClassRequestReconciler) getDefaultCephBlockPoolSpec() rookCephv1.NamedBlockPoolSpec {
+	return rookCephv1.NamedBlockPoolSpec{
+		PoolSpec: rookCephv1.PoolSpec{
+			EnableRBDStats: true,
+			FailureDomain:  r.storageCluster.Status.FailureDomain,
+			Replicated: rookCephv1.ReplicatedSpec{
+				Size:                     3,
+				ReplicasPerFailureDomain: 1,
+				TargetSizeRatio:          .49,
+			},
+		},
+	}
+}
```

Comment:
By design, we are not and should not create new block pools as a result of a new request. We cannot as we do not know the user's desired setting for the new block pool (which represents a new storage profile).

Please remove this code

Reply:
That's literally what the default block pool is for, but sure, done.

Reply:
> That's literally what the default block pool is for, but sure, done.

I think you misunderstood me. I agree with your statement, but that is also my point. The default block pool is already there. we should not start creating additional "default" block pools, we should utilize the one that already exists.